_posts/2025-02-27-use-desctop-visual-studio-code.adoc

```diff
@@ -0,0 +1,108 @@
+---
+title: "Develop in local Visual Studio Code using Eclipse Che as a remote Cloud Development Environment"
+layout: post
+author: Vitaliy Gulyy
+description: >-
+   Use your local Visual Studio Code as an editor for a Che workspace
+categories: []
+keywords: ['Visual Studio Code', 'Eclipse Che editor']
+slug: /@vgulyy/use-desktop-visual-studio-code
+---
+
+== Introduction
+
+Eclipse Che provides developers with a web version of the link:https://github.com/microsoft/vscode[Visual Studio Code OSS] editor, which includes most of the desktop version features. The transition to cloud development can feel quite unusual for developers, but Eclipse Che has a feature that will help you feel right at home immediately.
+With link:https://code.visualstudio.com/docs/editor/command-line[Visual Studio Code Command Line Interface (CLI)] you can create a tunnel that allows you to connect your familiar local Visual Studio Code to your workspace.
+
+== Connecting Visual Studio Code desktop to a new workspace
+
+You can connect your local Visual Studio Code to a new workpace with a few clicks.
+First, create a workspace on the Che Dashboard and choose Visual Studio Code (desktop) as your editor.
+
+image::/assets/img/use-desktop-visual-studio-code/select-editor.png["Create Workspace"]
+{nbsp} +
```

Comment:
@deerskindoll @vitaliy-guliy if you would like to have image descriptions those should be a separate entries see https://github.com/eclipse-che/blog/blob/main/_posts/2024-11-26-in-kubernetes-we-trust.adoc?plain=1#L22

Reply:
Honestly, I would leave the images as they are, without description.

Reply:
that's perfectly fine with me. it's just something I noticed and wanted to point out.